### DISCUSSION TITLE
luasnip

### DISCUSSION THREAD

**Discussion by harryhanYuhao** [neoclide/coc.nvim#4477] [Extension request]
As far as I know, there are no integration with LuaSnip for CoC. (Luasnip is not mentioned in coc-source).

Is there intended lua-snip integration in the future? Or if I missed anything, please remind and redirect me to informations as using CoC with luasnip.

**Comment by chemzqm** [comment]
I don't use LuaSnip as we have coc-snippets which provide UltiSnips snippets support, search npm or checkout `:h coc-api-vim-source` for create a custom vim source.
